Source/Particles/ElementaryProcess/QEDPairGeneration.H

```diff
@@ -119,7 +124,7 @@ public:
     * @param[in] engine random number generator engine
     */
     template <typename DstData, typename SrcData>
-    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE
+    AMREX_GPU_DEVICE AMREX_FORCE_INLINE
```

Comment:
I made this change because the cuda tests failed due to calling the device only function `chi_photon` from within this functor.

I think this change is ok because these QED transform functors are only called on the device.

Reply:
I agree with you that this change is fine.